incubator/harbor/LICENSE

```diff
@@ -0,0 +1,14 @@
+Copyright 2018 Paul Czarkowski (pczarkowski@pivotal.io)
```

Comment:
Everything in the charts repository is licensed under the top level license file. Can you remove this one? That file is at https://github.com/kubernetes/charts/blob/master/LICENSE